setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-version = '3.0.0'
+version = '3.0.1'
```

Comment:
I'll bump the version number when doing a release

Reply:
Sure